code/go/internal/validator/semantic/types.go

```diff
@@ -21,6 +23,74 @@ const dataStreamDir = "data_stream"
 
 type fields []field
 
+type runtime struct {
```

Comment:
Nit. This is a field, not a runtime :)
```suggestion
type runtimeField struct {
```